tests/unit/models/search.js

```diff
@@ -0,0 +1,287 @@
+describe("Search Model", function() {
+  var search = {}, journalResponse = {}, articleResponse = {};
+
+  beforeEach(function() {
+    search = browzine.search;
+
+    journalResponse = {
+      "data": [{
+        "id": 10292,
+        "type": "journals",
+        "title": "New England Journal of Medicine (NEJM)",
+        "issn": "00284793",
+        "sjrValue": 14.619,
+        "coverImageUrl": "https://assets.thirdiron.com/images/covers/0028-4793.png",
+        "browzineEnabled": true,
+        "browzineWebLink": "https://browzine.com/libraries/XXX/journals/10292"
+      }]
+    };
+
+    articleResponse = {
+      "data": {
+        "id": 55134408,
+        "type": "articles",
+        "title": "New England Journal of Medicine reconsiders relationship with industry",
+        "date": "2015-05-12",
+        "authors": "McCarthy, M.",
+        "inPress": false,
+        "availableThroughBrowzine": true,
+        "startPage": "h2575",
+        "endPage": "h2575",
+        "browzineWebLink": "https://browzine.com/libraries/XXX/journals/18126/issues/7764583?showArticleInContext=doi:10.1136/bmj.h2575"
+      },
+      "included": [{
+        "id": 18126,
+        "type": "journals",
+        "title": "theBMJ",
+        "issn": "09598138",
+        "sjrValue": 2.567,
+        "coverImageUrl": "https://assets.thirdiron.com/images/covers/0959-8138.png",
+        "browzineEnabled": true,
+        "browzineWebLink": "https://develop.browzine.com/libraries/XXX/journals/18126"
+      }]
+    };
+  });
+
+  afterEach(function() {
+
+  });
+
+  it("search model should exist", function() {
+    expect(search).toBeDefined();
+  });
+
+  it("should not enhance a search result without a document", function() {
+    var scope = {};
+
+    expect(search.shouldEnhance(scope)).toEqual(false);
+  });
+
+  it("should detect whether a journal search result can be enhanced", function() {
```

Comment:
Just a wording nitpick here, but since we're only testing the positive case in this test and the two below, could we rename the `it` message to something that conveys that? Such as:

`it("should choose to enhance a journal article with an ISSN")`